verbit/streaming_client.py

```diff
@@ -190,57 +203,148 @@ def _connect_websocket(self, media_config: MediaConfig, response_types: Response
         :param response_types: a bitmask Flag denoting which response type(s) should be returned by the server
         """
 
-        # build websocket url
+        # build WebSocket url
         ws_url = self.ws_url + self._get_ws_connect_query_string(media_config=media_config, response_types=response_types)
 
-        try:
+        # create WebSocket instance
+        self._ws_client = WebSocket(enable_multithread=True)
+
+        # set WebSocket client timeout
+        self._ws_client.timeout = 20.0
+
+        def _retry_http_error_predicate(retry_ex: WebSocketBadStatusException):
```

Comment:
not sure this is a predicate as well :)

```suggestion
        def _is_error_retryable(ex: WebSocketBadStatusException):
```

Reply:
I also prefer `_is_error_retryable` or something like `should_retry_http_error`